operation/fhir-operation-cpg/README.md

```diff
@@ -0,0 +1,14 @@
+# fhir-operation-cpg
+
+This is an optional module that provides a basic implementation of CQL-related operations found at present only in the [Clinical Practice Guidelines Implementation Guide](http://build.fhir.org/ig/HL7/cqf-recommendations/index.html) and upcoming FHIR R5 specifications. This includes the following:
+
+* /$cql
+* /Library/$evaluate
+* /Library/[id]/$evaluate
+
+These operations are assumed to interact with the local FHIR server for data, content, and terminology. If needed, terminology can be configured to be remote as described in the Terminology Guide. 
+
+## TODO
+
+* [ ] Implement prefetchData
+* [ ] Implement remote endpoints for data, content, and terminology
```

Comment:
how are these todos tracked? are there corresponding issues?

Reply:
I was not planning on implementing these additional pieces of function right now. I left them as TODOs in the README so that anyone that was picking up the work would understand that there was outstanding work to be done. It wasn't clear to me that you guys would want these issues in your backlog or not. I can certainly log them if you think it is important.

Reply:
https://github.com/IBM/FHIR/issues/2618